sections/footer.liquid

```diff
@@ -54,10 +54,11 @@
                 {%- when 'image' -%}
                   <div class="footer-block__details-content footer-block-image {{ block.settings.alignment }}">
                     {%- if block.settings.image != blank -%}
-                      {%- assign image_size = block.settings.image_width | append: 'x' -%}
+                      {%- assign image_width = block.settings.image_width -%}
+                      {%- assign image_width_2x = image_width | times: 2 | at_most: 5760 -%}
```

Comment:
That part is slightly annoying. `image_url: width: X` throws an error if the width > 5760. I don't have a lot of context on if this is even possible in this theme. But it's something I got worried about. 

![image](https://user-images.githubusercontent.com/4990691/144094066-b70b11fa-f8d3-4db0-949e-27ee2bf10b26.png)